### PR TITLE
scripts: refactored into multiple scripts

### PR DESCRIPTION
**Pull request by Hashfyre**
splits update.sh into multiple scripts.

### CODE REVIEW

File: scripts/cluster-auth.sh [hunk 1]
```diff
@@ -1,49 +1,6 @@
 #!/bin/bash
 set -euo pipefail
 
-# globals
```
Comment (ace03uec) [inline scripts/cluster-auth.sh:4]: Is the permission change expected ? Do you need to do a `+x` ?

**Review by ace03uec — APPROVED** [review]
Are we just splitting the functionality across pages ?

**Review by ace03uec — APPROVED** [review]
LGTM.

File: scripts/cluster-auth.sh (hunk 1 above)
Comment (Hashfyre) [reply scripts/cluster-auth.sh:4]: removed the +x permission.

Comment (Hashfyre) [reply scripts/cluster-auth.sh:4]: removed the `+x`, not needed.

**Pull request merged**